spec/features/topic_spec.rb

```diff
@@ -114,6 +114,7 @@
 
     within '.full-error-list' do
       expect(page).to have_content "Path must be present and start with '/service-manual/'"
+      expect(page).to have_content "Title can't be blank"
```

Comment:
It'd be nice to test this in the model spec. There _should_ be similar assertions on the existing validations in there..